### DISCUSSION TITLE
Curso 3054- Fullstack.NET - Erro ao aplicar migrations

### DISCUSSION THREAD

**Discussion by ialexandres** [balta-io/forum#688] [Tira Dúvidas] [answered]
Boa tarde! Estou com um problema ao aplicar a primeira Migrations.  Erro conforme imagem:

![image](https://github.com/user-attachments/assets/af3f44e5-a218-4982-a6bc-73d7287c5256)

Obrigado.

**Comment by henriqueweege** [comment, marked as answer]
oi, @ialexandres ! Tudo certinho?
pelo que vi no log, parece ter um "w" faltando em "password" ;)
valeeeu!
